indexer-app/src/cmr/indexer/data/concepts/service.clj

```diff
@@ -125,6 +135,7 @@
   [services]
   {:has-formats (boolean (some has-formats? services))
    :has-transforms (boolean (some has-transforms? services))
+   :has-combine (boolean (some has-combine? services))
    :has-variables (boolean (some has-variables? services))
```

Comment:
Can we change this to has-concatenate ?  Concatenate has a specific meaning and it also aligns better to the actual UMM-S metadata field.